content/docs/tutorials/interactivity/advanced-interactivity.md

```diff
@@ -0,0 +1,216 @@
+---
+$title: Improving the interactivity
+$order: 2
+toc: true
+---
+
+[TOC]
+
+The starter code provides a pretty bare user experience. There are a couple ways we can improve it:
+
+- Add an indicator that displays the current slide and total number of slides.
+- When a user selects a different shirt color, change the image carousel to show images of shirts in the selected color.
+
+Prior to the introduction of the `<amp-bind>` component, adding features like these weren't possible. Let's get a hands-on experience with `<amp-bind>` and add these new features to our sample code!
+
+## Install the `<amp-bind>` extension
+
+`<amp-bind>` is a new AMP component that affords custom interactivity via data binding and JS-like expressions. To use `<amp-bind>`, you must install it in the page.
+
+Open the `static/index.html` file, and add the following script to the list of AMP components in the `<head>` section of the page:
```

Comment:
Add link to https://github.com/googlecodelabs/advanced-interactivity-in-amp/blob/master/static/index.html